routes/trips/tripRoutes.js

```diff
@@ -9,6 +9,7 @@ router.use(authenticateToken);
 
 // routes
 router.get('/', tripCtrl.getTrips);
+router.get('/', userToTrip, tripCtrl.getJoinedTrips);
```

Comment:
we can't have two .gets to the same '/' route. 
We'll need to give it a different more descriptive route like '/users/:tripid' or something